store/json.go

```diff
@@ -23,7 +23,7 @@ const (
 	LockExtension = ".lock"
 
 	// DefaultLockTimeout - lock timeout in milliseconds
-	DefaultLockTimeout        = 10000 * time.Millisecond
+	DefaultLockTimeout        = 30000 * time.Millisecond
```

Comment:
can we name this `DefaultLockTimeoutLinux`?

Reply:
I will add another var name "DefaultLockTimeoutLinux", since 
The DefaultLockTimeout is being used by telemetry